include/localization.js

```diff
@@ -472,7 +472,6 @@ module.exports = function LocalizationModule(pb) {
      * @param {Function} cb
      */
     Localization.init = function(cb) {
-        var supportedLocales = [];
```

Comment:
Some of the changes in this PR confuse me. This one for instance has already been merged? (https://github.com/careerbuilder/pencilblue/pull/154)